vuejs/src/org/jetbrains/vuejs/intentions/expandVModel/VueExpandVModelIntention.kt

```diff
@@ -0,0 +1,68 @@
+package org.jetbrains.vuejs.intentions.expandVModel
+
+import com.intellij.lang.javascript.intentions.JavaScriptIntention
+import com.intellij.openapi.application.WriteAction
+import com.intellij.openapi.command.CommandProcessor
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.xml.XmlAttribute
+import com.intellij.psi.xml.XmlElementType
+import org.jetbrains.vuejs.codeInsight.attributes.VueAttributeNameParser
+import org.jetbrains.vuejs.codeInsight.attributes.VueAttributeNameParser.*
+import org.jetbrains.vuejs.codeInsight.attributes.VueAttributeNameParser.VueAttributeKind.DIRECTIVE
+import org.jetbrains.vuejs.codeInsight.tags.VueElementDescriptor
+import org.jetbrains.vuejs.index.isVueContext
+
+class VueExpandVModelIntention : JavaScriptIntention() {
+    override fun getFamilyName(): String = "Expand v-model"
+    override fun getText(): String = this.familyName
+    private val validModifiers = setOf("lazy", "number", "trim")
+
+    override fun isAvailable(project: Project, editor: Editor?, element: PsiElement): Boolean {
+        return element.node.elementType == XmlElementType.XML_NAME &&
+                element.parent?.node?.elementType == XmlElementType.XML_ATTRIBUTE &&
+                element.parent.isValid &&
```

Comment:
I think you should add a check for the component descriptor here as well.

Reply:
Is it really needed? I don't use it anywhere.. How should I check it, in case?

Reply:
You look for it in the `invoke` logic:
```
 val componentDescriptor = componentTag.descriptor as? VueElementDescriptor ?: return
```
And if the descriptor is missing nothing happens, such behavior might be confusing for users.